packages/select-rich/src/LionSelectRich.js

```diff
@@ -129,9 +149,18 @@ export class LionSelectRich extends OverlayMixin(
     return this.formElements.findIndex(el => el.active === true);
   }
 
+  get scrollTarget() {
+    return this._overlayContentNode.scrollTarget || this._overlayContentNode;
+  }
+
   set activeIndex(index) {
     if (this.formElements[index]) {
-      this.formElements[index].active = true;
+      const el = this.formElements[index];
+      el.active = true;
+
+      if (!isInView(this.scrollTarget, el)) {
+        el.scrollIntoView({ behavior: 'smooth', block: 'nearest' });
```

Comment:
The options have limited support:

https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView

Reply:
yeah it's not as smooth but it gives the same end result 👍